frontend/src/locales/zh-CN.json

```
@@ -702,20 +701,18 @@
           "title": "禁止提交包含错误告警的工单",
           "description": "启用后，只有在预览阶段不包含任何“错误”级别告警的工单才能被提交。"
         },
-        "database-change-mode": {
-          "self": "数据库变更模式",
-          "description": "控制用户如何变更数据库",
-          "issue-mode": {
-            "self": "工单模式（管控）",
-            "description": "使用工单来提交，审核，发布，版本化数据库变更，适合具有访问控制和 CI 和 CD 工作流的团队。"
+        "default-landing-page": {
+          "self": "默认着陆页",
```

maybe we should use...

```suggestion
          "self": "默认首页",
```